proxy/agent.go

```diff
@@ -78,6 +78,12 @@ var (
 	}
 )
 
+const (
+	// MaxAborts is the maximum number of cascading abort messages to buffer.
+	// This should be the upper bound on the number of proxies available at any point in time.
+	MaxAborts = 10
```

Comment:
Ultimately, it's the proxy that decides how many processes to keep around. Envoy has up to 3 .